paper-api/src/main/java/org/bukkit/entity/Villager.java

```diff
@@ -391,4 +391,15 @@ static Profession[] values() {
      * reputation regardless of its impact and the player associated.
      */
     public void clearReputations();
+
+    /**
+     * Updates the demand for Villager offers.
+     * This is used to calculate price changes for regularly traded offers.
+     */
+    public void updateDemand();
+
+    /**
+     * Restocks all offers for the Villager.
+     */
+    public void restock();
```

Comment:
Can be good mention this also call the internal updateDemand and also mention this can trigger an VillagerReplenishTradeEvent